sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -251,9 +251,9 @@ object FunctionRegistry {
     expression[Log10]("log10"),
     expression[Log1p]("log1p"),
     expression[Log2]("log2"),
-    expression[Log]("ln"),
+    expression[Log]("ln", true),
     expression[Remainder]("mod", true),
-    expression[UnaryMinus]("negative"),
+    expression[UnaryMinus]("negative", true),
```

Comment:
I think you should manually handle the aliases in `UnaryMinus`. The output column name will still be based on `override def sql: String = s"(- ${child.sql})"`.

Reply:
OK. I will open another PR to manually handle the aliases. This PR is used to add more powerful guarantees so that aliases outputed by built-in functions are correct.

Reply:
@HyukjinKwon This alias should be `negative` or `-` ?

Reply:
Could we use `negative(1)` as alias ?

Reply:
Yes, I think we can use `negative(1)` as alias. Does that work if we check the tag in override `sql`?

Reply:
See https://github.com/apache/spark/pull/27901/files#diff-1516b10738479bbe190fb4e239258473R467-R470

Reply:
@HyukjinKwon There are so many kinds of rewrites of SQL methods. it seems difficult to check.